spec/support/factory_girl.rb

```diff
@@ -15,4 +15,11 @@
     feel 'Cold'
   end
 
+  factory :review do
+    comment 'Nice weather'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.